src/pages/Checkout/Checkout.js

```diff
@@ -0,0 +1,42 @@
+import styled, { css } from 'styled-components';
+
+export const Page = styled.div`
+  max-width: ${props => props.title === 'checkout' && '37.5rem'};
+  width: ${props => props.title === 'commun' && '60.625rem'};
+  margin: 0 auto;
+`;
+
+export const Title = styled.div`
+  font-size: ${props => (props.size === 'big' ? '30px' : '20px')};
+  padding: ${props =>
+    props.size === 'big' ? '40px 0 20px 0' : '60px 0 20px 0'};
+`;
+
+export const CheckBox = styled.input.attrs({ type: 'checkbox' })``;
+
+export const AgreeBtn = styled.button`
+  margin: ${props => props.type === 'point' && '0 10px'};
```

Comment:
```suggestion
${props => props.type === "point" && css`
margin: 0 10px 
`}
 
```
type 이 point가 아닐때를 대비해주셔야 할 것 같습니다